src/gemini_client.sh

```
@@ -0,0 +1,263 @@
+    echo "$response"
+}
+
+# メイン実行関数
+generate_commit_message() {
+    local diff_content="$1"
+    local file_analysis="$2"
+    
+    log_info "Gemini CLIを使用してコミットメッセージを生成中..."
+    
+    # 設定を読み込み
+    local config=$(load_config)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)